submissions/crealime/html-css-popup/css/main.css

```diff
@@ -5,6 +5,11 @@
   font-style: normal
 }
 
+:root {
+  --brd-visible: solid 1px #b4b4b4;
+  --brd-hidden: solid 1px transparent
```

Comment:
Suggestions about naming: `primary-border` and `transparent-border` looks better IMO. And don't be afraid of long names, it sounds more clear

Reply:
Changed. Thank you )

Reply:
Less button has to be hidden, because this is the same button in menu..